### PR TITLE
Move config file to cmd so it can be set when running the container

### PR DESCRIPTION
**Pull request by MartinLeedotOrg**
Signed-off-by: Martin Lee <martin@billforward.net>

### CODE REVIEW

**Comment by MartinLeedotOrg** [comment]
@brian-brazil

File: Dockerfile [hunk 1]
```diff
@@ -12,4 +12,5 @@ WORKDIR /
 RUN mkdir /config
 
 ONBUILD ADD config.yml /config/
-ENTRYPOINT [ "java", "-jar", "/cloudwatch_exporter.jar", "9106", "/config/config.yml" ]
+ENTRYPOINT [ "java", "-jar", "/cloudwatch_exporter.jar", "9106"]
```
Comment (brian-brazil) [inline Dockerfile]: Why move the config and not the port?

Comment (MartinLeedotOrg) [reply Dockerfile]: Port mapping can be changed easily with `-p 91xx:9106`

Can move it in to the CMD if you'd like though.

Comment (brian-brazil) [reply Dockerfile]: Makes sense

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**